source/qml/Samples/QmlVisualizer/AdaptiveCardQml/TextInputRender.qml

```diff
@@ -0,0 +1,164 @@
+﻿import "AdaptiveCardUtils.js" as AdaptiveCardUtils
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.3
+import AdaptiveCards 1.0
+
+
+Column{
+    id:_inputText   
+    
+    property int _spacing
+    property color _labelColor
+    property bool _isRequired: false
+    property string _mEscapedLabelString
+    property string _mEscapedErrorString
+    property string _mEscapedPlaceHolderString
+    property string _mEscapedValueString
+    property bool _supportsInterActivity
```

Comment:
I think we can just check SupportsInteractivity in C++ layer

Reply:
i need it for loader element ,
this is just a flag to indicate whether we have any action button or not , based on that i am using my loaders